.github/workflows/performance_testing.yml

```diff
@@ -12,9 +12,9 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: Pull Docker Test Container
-        run: docker pull registry.opensuse.org/home/schoolguy/branches/opensuse/templates/images/tumbleweed/containers/opensuse/cobbler-github-ci:latest
+        run: docker pull registry.opensuse.org/systemsmanagement/cobbler/github-ci/containers/cobbler-test-github:suma-43
```

Comment:
Not sure if it's correct, I see in https://github.com/openSUSE/cobbler/pull/61 `suma-43` tag used in all 3 changed files, but here this is the only one refering to `suma-43` tag while the rest refer to `suma-head`

Reply:
You are right. This is now fixed. Thanks!